packages/safe-apps-sdk/dist/src/types/sdk.d.ts

```diff
@@ -22,6 +22,18 @@ export declare type GetBalanceParams = {
 export declare type SignMessageParams = {
     message: string;
 };
+interface TypedDataTypes {
+    name: string;
+    type: string;
+}
+export declare type TypedMessageTypes = {
+    [key: string]: TypedDataTypes[];
+};
+export declare type SignTypedMessageParams = {
+    domain: Record<string, any>;
```

Comment:
I think we can add a more specific type for the domain

```
export interface TypedDataDomain {
    name?: string;
    version?: string;
    chainId?: BigNumberish;
    verifyingContract?: string;
    salt?: BytesLike;
};
```

taken from https://github.com/ethers-io/ethers.js/blob/608864fc3f00390e1260048a157af00378a98e41/packages/abstract-signer/src.ts/index.ts#L25

The description of the fields can be found in the [EIP-712 Standard](https://eips.ethereum.org/EIPS/eip-712) in the `Definition of domainSeparator` section

Reply:
Updated, thanks!